microsoft-edge/toc.yml

```
@@ -1033,89 +1050,166 @@
+
+            - name: Policies for extensions
+              items:
+                - name: App Developer Agreement Addendum for Microsoft Edge program users
+                  href: extensions-chromium/store-policies/ada-addendum.md
+                  displayName: policy, policies  # synonym lookup
+
+                - name: Developer policies for the Microsoft Edge Add-ons store
+                  href: extensions-chromium/store-policies/developer-policies.md
+                  displayName: policy  # synonym lookup
+
+# -------------------------------------
+        - name: Publish an extension
           items:
-            - name: Register as a Microsoft Edge extension developer
-              href: extensions-chromium/publish/create-dev-account.md
+            - name: Register as an individual developer
+              items:
+                - name: Register as a Microsoft Edge extension developer
+                  href: extensions-chromium/publish/create-dev-account.md
+
+            - name: Register as an enterprise developer
+              items:
+#               - name: Register as a Microsoft Edge extension developer for Company Account
+#                 href:
+
+                - name: Verify your company account information
+                  href: extensions-chromium/publish/verify-microsoft-edge-program.md
 
-            - name: Verify your company account information
-              href: extensions-chromium/publish/verify-microsoft-edge-program.md
+                - name: Add users to the Microsoft Edge program
+                  href: extensions-chromium/publish/aad-account.md
 
             - name: Publish a Microsoft Edge extension by using a GitHub account
               href: extensions-chromium/publish/github.md
 
-            - name: Add users to the Microsoft Edge program
-              href: extensions-chromium/publish/aad-account.md
-
             - name: Publish a Microsoft Edge extension
               href: extensions-chromium/publish/publish-extension.md
 
-            - name: Extension hosting
-              href: extensions-chromium/publish/hosting-and-updating.md  # moved from /enterprise/
-
-            - name: Update a Microsoft Edge extension
-              href: extensions-chromium/publish/update-extension.md
+#           - name: Extension sharing offline
+#             href:
 
-            - name: Automatically update extensions in Microsoft Edge
-              href: extensions-chromium/publish/auto-update.md  # moved from /enterprise/
+            - name: Extension hosting
+              href: extensions-chromium/publish/hosting-and-updating.md
 
             - name: Submission states for extensions in the Microsoft Edge Add-ons website
               href: extensions-chromium/publish/submission-states.md
 
+            - name: Manage account settings
+              href: extensions-chromium/publish/manage-settings.md
+
+            - name: Alternative ways to distribute extensions
```

```suggestion
            - name: Alternative ways to distribute an extension
```
the task & use-case is: user wants an alt way to distribute _an_ extension, not multiple extensions.  shift from abstract, system-oriented doc'n to concrete, task-oriented doc'n.

change title in all the usual spots.
add to PR change list.